lib/fork.js

```diff
@@ -36,7 +36,9 @@ module.exports = (file, opts, execArgv) => {
 		} : false
 	}, opts);
 
-	const ps = childProcess.fork(path.join(__dirname, 'test-worker.js'), [JSON.stringify(opts)], {
+	var args = [JSON.stringify(opts), opts.color ? '--color' : '--no-color'];
```

Comment:
I'll add the `esnext: true` option when https://github.com/avajs/ava/pull/1154 lands (hopefully in the next few days), as it's the last piece of non-ES2015ified code.